.coveragerc

```diff
@@ -4,6 +4,7 @@ parallel = True
 source = twisted
 omit =
     .tox/*/tmp/_trial_temp/*
+    */twisted/internet/gtk2reactor.py
```

Comment:
FYI: adding this here because I don't want to play games to get coverage in a module which is now deprecated and has abysmal test coverage. The code probably works fine, and so I don't want to rip it out with no warning, but I don't want to waste any more effort on a library that had its last release 10+ years ago in favor of one that is actively used today